File: MAP/Interpretor2/src/Model/ConstExp.java
package Model;

public class ConstExp extends Exp {
	Integer n;
	public ConstExp(Integer i)
	{
		this.n=i;
		
	}

	@Override
	public int eval(MyIDictionary<String, Integer> tbl,MyIDictionary<Integer,Integer> heap) throws Exception {
		return this.n;
	}
	@Override
	public String toString()
	{
		return this.n.toString();
	}
	

}

File: Interpreter C#/Interpreter/ConstExp.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Interpreter
{
    namespace Model
    {
        class ConstExp:Exp
        {
            int number;

            public ConstExp(int nr)
            {
                this.number = nr;
            }

            public override int eval(MyIDictionary<string, int> tbl)
            {
                return this.number;
            }

            public override string ToString()
            {
                return this.number.ToString();
            }
        }
    }
}
